### DISCUSSION TITLE
Add zoom function to image preview

### DISCUSSION THREAD

**Discussion by Dumpstar56** [directus/directus#14953] [Feature Requests]
I would love to have a zoom function for the image preview of uploaded images when the file is linked in a m2o or m2m relationship in the db - or in the file directory without having to go into edit mode. So something like this:
![zoom](https://user-images.githubusercontent.com/94397183/183448890-cc074c18-f2e6-4969-b93c-a2e34473c976.jpg)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
